nni/algorithms/compression/pytorch/pruning/net_adapt_pruner.py

```diff
@@ -149,7 +149,7 @@ def _update_config_list(self, config_list, op_name, sparsity):
                 return config_list_updated
 
         # if op_name is not in self._config_list_generated, create a new json item
-        if self._base_algo in ['l1', 'l2']:
+        if self._base_algo in ['l1', 'l2', 'fpgm']:
```

Comment:
a little confused. why ADMMPruner need to update `_projection` function, but net_adapter_pruner does not need such modification?